build.zig

```diff
@@ -36,7 +36,7 @@ pub fn build(b: *Builder) !void {
     const docs_step = b.step("docs", "Build documentation");
     docs_step.dependOn(&docgen_cmd.step);
 
-    const test_step = b.step("test", "Run all the tests");
+    const toolchain_step = b.step("test-toolchain", "Run the tests for the toolchain");
```

Comment:
Let's please keep "test" to do the same thing it always did, but we can additionally introduce `test-toolchain` to be the subset that you have picked out.

Reply:
`test` is `toolchain_step`+`std_step`+`docs_step`, so it should still do the same thing is always did.

Reply:
Ahh missed that before. Thanks!